examples/dlp/cloud_function_example/README.md

```diff
@@ -62,7 +64,7 @@ Create the export.
 export PROJECT_ID=$(gcloud config get-value project)
 
 gcloud logging sinks create $LOG_EXPORT_NAME \
-  pubsub.googleapis.com/projects/$PROJECT_ID/topics/$LOG_EXPORT_TOPIC_NAME \
+  $LOG_EXPORT_TOPIC_NAME pubsub.googleapis.com/projects/$PROJECT_ID/topics/$LOG_EXPORT_TOPIC_NAME \
```

Comment:
I think you added the $LOG_EXPORT_TOPIC_NAME by mistake?

Docs say it should be `gcloud logging sinks create SINK_NAME DESTINATION`. 
The destination should only be the fully qualified destination topic?

Reply:
@tims That's what I thought as well until I tried to run the command a second time. 
I get this error when I tried to use DESTINATION a second time ```is the subject of a conflict: Sink DESTINATION already exists```

![image](https://user-images.githubusercontent.com/19396704/73668941-e374a300-4663-11ea-82db-e97c73d295b1.png)